recipes/obitools/meta.yaml

```diff
@@ -1,78 +1,49 @@
 {% set name = "OBITools" %}
-{% set version = "1.2.13" %}
-{% set sha256 = "efc7b583a44f435f0663f9d67759a0e3d415ba1896e6132198cb135e35c34873" %}
+{% set version = "3.0.0-beta3" %}
```

Comment:
@lecorguille do we need a beta version? I would rather wait for a stable version to include in Bioconda.

Reply:
One of our users requested for this tool.
And since 1 year now, we prioritize Conda installation.

If it's an issue, I will push it on a temp channel.

Reply:
I'd suggest pushing it to a temp channel.